{"content": {"suites": {"0": {"status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 2, "total_error": 0}, "tests": {"0": {"status": "FAIL", "message": "       AssertionError: login failed\n       assert 'https://candymapper.com/m/account' == ('https://candymapper.com/m/account', 'Login failed')\n        +  where 'https://candymapper.com/m/account' = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"c72ef72442afae7b7cb82fe0fc86dc96\")>.current_url\n        +    where <selenium.webdriver.chrome.webdriver.WebDriver (session=\"c72ef72442afae7b7cb82fe0fc86dc96\")> = <test.test_SignIn.TestSignin testMethod=test_login>.driver\n        +  and   ('https://candymapper.com/m/account', 'Login failed') = SuccessUrl.login_pass\n       AssertionError: Code error\n       assert 1 == 0\n", "test_name": "test_login", "rerun": "0"}, "1": {"status": "FAIL", "message": "   selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//*[@id=\"4\"]\"}\n     (Session info: chrome=127.0.6533.88); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\n   Stacktrace:\n   #0 0x64e0fb0f571a <unknown>\n   #1 0x64e0fadc6640 <unknown>\n   #2 0x64e0fae15c0b <unknown>\n   #3 0x64e0fae15ef1 <unknown>\n   #4 0x64e0fae59b64 <unknown>\n   #5 0x64e0fae3890d <unknown>\n   #6 0x64e0fae5708a <unknown>\n   #7 0x64e0fae38683 <unknown>\n   #8 0x64e0fae08d71 <unknown>\n   #9 0x64e0fae097de <unknown>\n   #10 0x64e0fb0bd2ab <unknown>\n   #11 0x64e0fb0c1242 <unknown>\n   #12 0x64e0fb0aa665 <unknown>\n   #13 0x64e0fb0c1dd2 <unknown>\n   #14 0x64e0fb08f2af <unknown>\n   #15 0x64e0fb0e4eb8 <unknown>\n   #16 0x64e0fb0e5090 <unknown>\n   #17 0x64e0fb0f44ec <unknown>\n   #18 0x7de493e94ac3 <unknown>\n       AssertionError: Code error\n       assert 1 == 0\n", "test_name": "test_wrong_login", "rerun": "0"}}, "suite_name": "test_SignIn.py"}}}, "date": "August 01, 2024", "start_time": 1722511517.5073497, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "0", "fail": "2", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "2"}